src/main/scala/codacy/docker/api/duplication/Duplication.scala

```diff
@@ -0,0 +1,11 @@
+package codacy.docker.api.duplication
+
+//This was declared in the seed along with the other two case classes.
```

Comment:
Can you explain this better? I do not understand what you mean

Reply:
I didn't find usages of the DuplicationRequest case class in the worker, duplication seed or in the duplication tools. I was asking if it was something left from a previous implementation of duplication that was forgotten in the duplication seed code. 
Since then I double-checked it and found that there is a slightly different DuplicationRequest defined in codacy-plugins that is used in the worker and also another one defined in the worker at pluginsEngineComponent.api.DuplicationRequest that has a different purpose.
Do you know if this case class is still being used elsewhere?